pkg/secrets/secrets.go

```diff
@@ -38,10 +39,11 @@ func init() {
 // Init initializes the command and other options of the secrets package. Since
 // this package is used by the 'config' package to decrypt itself we can't
 // directly use it.
-func Init(command string, arguments []string, timeout int, maxSize int) {
+func Init(command string, arguments []string, termTimeout int, killTimeout int, maxSize int) {
 	secretBackendCommand = command
 	secretBackendArguments = arguments
-	secretBackendTimeout = timeout
+	secretBackendTermTimeout = termTimeout
+	secretBackendKillTimeout = killTimeout
```

Comment:
I think we should check that termTimeout < killTimeout and log a warning if not. This should also be done in the `secret` command (see https://github.com/DataDog/datadog-agent/blob/master/pkg/secrets/info.go).